lib/runtime/test_helpers.go

```diff
@@ -99,6 +101,9 @@ func GetRuntime(ctx context.Context, runtime string) (
 		runtimeFilename = WESTEND_RUNTIME_V0929_FP
 		url = WESTEND_RUNTIME_V0929_URL
 	default:
+		if utils.PathExists(runtime) {
+			return runtime, nil
+		}
```

Comment:
supernit maybe move this up at the top of the function, so that:

- the directory `/tmp/gossamer/runtimes/` isn't created if we use a runtime file we specify
- if we have a file with the same name as the runtime string constant (i.e. `WESTEND_RUNTIME_v0929`), the file specified takes precedence over the hardcoded one

The comment above `If the runtime argument is not defined in the constants.go and is a valid` may have to be changed a bit.